src/services/itemMembership/index.ts

```diff
@@ -28,7 +28,7 @@ const plugin: FastifyPluginAsync = async (fastify) => {
   // schemas
   fastify.addSchema(common);
 
-  fastify.register(MembershipRequestAPI);
+  fastify.register(MembershipRequestAPI, { prefix: '/items/:itemId/memberships/requests' });
```

Comment:
As discussed offline, I think this hides the `:itemId` parameter. But we said that a bigger refactor is needed in order to have CORS at only 2 places (for the core api and for the apps API). This means we will have to move the cors instantiation up and extract the apps api from the item plugins.